modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase

```diff
@@ -1106,13 +1104,17 @@ definition {
 
 		Navigator.gotoNavItem(navItem = "Performance");
 
+		Pause(locator1 = "25000");
+
 		Click(
 			key_index = "4",
 			locator1 = "WorkflowMetrics#DASHBOARD_TOTAL_COMPLETED_ITEMS_BUTTON");
 
+		Refresh();
+
 		Click(locator1 = "WorkflowAllItems#WORKFLOW_INSTANCE_ID");
 
-		AssertElementPresent(
+		WaitForElementPresent(
 			key_status = "on Time",
 			locator1 = "WorkflowAllItems#DETAIL_PAGE_SLA_DEFINITION_RESOLVED");
```

Comment:
Even with that Pause, which it's no really good, depending on the CI behavior running the suites, the test may be a problem. increase the time of sla ending, after creating the blog, use waitForSla to check when the frame will have reached "on Time" and then finish(approve) before reaching overdue, then when checking the performance tab, you can click do "completed on time" card, then we will have ensured that the on time is being respected, so it will not be necessary to assert within the modal. Thank you!